cmd/build-init/main.go

```diff
@@ -147,6 +151,9 @@ func main() {
 		logger.Fatalf("error while processing the project descriptor: %s", err)
 	}
 
+	logger.Printf("Builder:\n Image: %s \n Name: %s \n Kind: %s ", *builderImage,
```

Comment:
The build may be executed without this info. In that case it shouldn't log this message.